src/btree/bt_split.c

```diff
@@ -787,7 +788,7 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session,
 static int
 __split_parent(WT_SESSION_IMPL *session, WT_REF *ref, WT_REF **ref_new,
     uint32_t new_entries, size_t parent_decr, size_t parent_incr,
-    int exclusive, int ref_discard, uint64_t *split_genp)
+    int exclusive, uint64_t *split_genp)
```

Comment:
@michaelcahill: another nice thing about this change -- I think we no longer need to pass `parent_decr` to `__split_parent`, and can remove it from `__wt_split_insert` and `__wt_split_multi`.